Base/res/html/misc/new-tab.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>New Tab</title>
+    <style>
+        body {
+            display: flex;
+            align-items: center;
+            justify-content: center;
+        }
+
+        main {
+            text-align: center;
+            display: block;
+            width: 100%;
+            max-width: 400px;
+        }
+
+        img {
+            image-rendering: pixelated;
+        }
+
+        input[type=search] {
+            width: 100%;
+            padding: 5px;
+        }
+
+        #search-buttons {
+            display: flex;
+            justify-content: space-between;
+            align-items: center;
+        }
+
+        button:hover {
+            cursor: pointer;
+        }
+    </style>
+</head>
+<body>
+    <main>
+        <br>
+        <img src="/res/icons/32x32/app-browser.png" width="64" height="64"><br><br>
+        <form>
+            <input type="search" name="q" id="user_query"><br><br>
```

Comment:
```suggestion
            <input type="search" name="q"><br><br>
```

Reply:
> Likewise - style/access input[type=search] instead, **or be more specific than `input` with the id name**

Was not `user_query` specific enough?

Reply:
it's specific enough, but kinda pointless for form elements with a `name` :)

Reply:
I guess the point in this case is that people write code differently, there is no code style docs for html/css/js afaik and this was one of the suggested changes I could use so I picked the one I prefer... ;-)

Reply:
Fair enough, I don't feel strongly about it.